serverless.yml

```diff
@@ -0,0 +1,157 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
```

Comment:
I kept this file close to the [original template file](https://github.com/serverless/serverless-azure-functions#create-a-new-azure-function-app) generated using `sls create`. Should I clean it up instead?

Reply:
I think that you can clean it and add a link to the docs for the format of the file.